doc/manual/command-ref/conf-file.xml

```diff
@@ -29,13 +29,36 @@
     (i.e. <filename>/etc/nix/nix.conf</filename> on most systems), or
     <filename>$NIX_CONF_DIR/nix.conf</filename> if
     <envar>NIX_CONF_DIR</envar> is set.</para>
+
+    <para>Values loaded in that file are not forwarded to the nix-daemon. The
+    client assumes that the daemon has already loaded them.
+    </para>
+  </listitem>
+
+  <listitem>
+    <para>
+      For each folder in <envar>XDG_CONFIG_DIRS</envar>, separated by
+      <literal>:</literal>.
+
+      Load <filename><replaceable>folder</replaceable>/nix/nix.conf</filename>.
+    </para>
   </listitem>
 
   <listitem>
-    <para>The user configuration file
-    <filename>$XDG_CONFIG_HOME/nix/nix.conf</filename>, or
-    <filename>~/.config/nix/nix.conf</filename> if
-    <envar>XDG_CONFIG_HOME</envar> is not set.</para>
+    <para>
+      The user configuration file in
+      <filename>$XDG_CONFIG_HOME/nix/nix.conf</filename> where
+      <envar>XDG_CONFIG_HOME</envar> defaults to
+      <filename>$HOME/.config</filename>.
+    </para>
+  </listitem>
+
+  <listitem>
+    <para>
+      And finally, every files pointed to by
+      <envar>NIX_CONF_FILES</envar>, separated by <literal>:</literal>
+      if that environment variable exists.
```

Comment:
This is phrased awkwardly (it's the *contents* of NIX_CONF_FILE that's separated by `:`).